src/main/java/io/wisetime/connector/api_client/ApiClient.java

```diff
@@ -94,4 +95,13 @@
    * @throws IOException
    */
   SubscribeResult postedTimeSubscribe(SubscribeRequest subscribeRequest) throws IOException;
+
+  /**
+   * Removes a webhook on receiving posted time.
```

Comment:
This might be misread as we remove the webhook when posted time has been received. Can we use something like

> Deletes a posted time webhook.

Reply:
Ok, will update